.travis.yml

```diff
@@ -21,6 +21,7 @@ before_script:
 before_install:
   - "npm install -g npm@^4"
   - "google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 http://localhost &"
+bundler_args: --retry 0
```

Comment:
Is this travis specific or it's passed to lerna or something ? 
I can't find options on lerna or npm and travis ci references bundler_args with the ruby bundler